manifest_staging/deploy/infra/deployment-rbac.yaml

```diff
@@ -479,6 +479,7 @@ spec:
         args:
           - "--node=$(NODE_NAME)"
           - "--http-probe-port=8085"
+          - "--enableScaleFeatures=true"
```

Comment:
Can we remove this since it's true by default?

Reply:
I think we should keep it until the flag is supported so user's can just disable when they want to without looking at the docs.